libevm/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@
 set(sources
     ExtVMFace.cpp ExtVMFace.h
     Instruction.cpp Instruction.h
+    interpreter.h
```

Comment:
Lowercase?

Reply:
Because there is no Interpreter class there and it follows the convention of EVM-C implementation headers: evmjit.h, hera.h, interpreter.h.

Reply:
Ok, how about also creating a directory for it in that case?

Reply:
I was thinking about this, but that requires another PR. Changing and moving files in the same time makes reviews horrible.